### PR TITLE
Space out readme commands for clarity, and add a final message that p…

### PR DESCRIPTION
**Pull request by jamescgoertzen**
…rompts users to reset their terminal

### CODE REVIEW

File: exe/clio_setup [hunk 1]
```diff
@@ -64,4 +64,30 @@ STEPS.each do |klass|
...
+puts '           (   /                     \_}'
+puts '              |               /      |'
+puts '              ;     |         \      /'
+puts '               \/ ,/           \    |'
+puts '               / /~~|~|~~~~~~|~|\   |'
+puts '              / /   | |      | | `\ \ '
+puts '             / /    | |      | |   \ \ '
+puts '            / (     | |      | |    \ \ '
+puts '      |||| /,_)||| /__) ||| /__)|| /,_/ |||'
+puts '     """"""""""""""""""""""""""""""""""""""'
```
Comment (fimmtiu) [inline exe/clio_setup]: Goat looks like a horse. Could we shorten the muzzle a bit?

File: exe/clio_setup [hunk 2]
```diff
@@ -64,4 +64,30 @@ STEPS.each do |klass|
...
+puts "\r\r\rWelcome home."
+puts "\r\rPlease restart your terminal for the appropriate changes to take effect."
+puts "After restarting, navigate to the themis directory and run `overmind start` to start your local server."
+puts "You will then able to load Clio through `localhost:3000` in your browser."
```
Comment (fimmtiu) [inline exe/clio_setup]: Let's include the scheme.
```suggestion
puts "You will then able to load Clio through `http://localhost:3000/` in your browser."
```

File: exe/clio_setup [hunk 3]
```diff
@@ -64,4 +64,30 @@ STEPS.each do |klass|
...
+puts "The first time loading Clio can take a little while, as caches need to be built."
```
Comment (fimmtiu) [inline exe/clio_setup]: Let's be more specific!
```suggestion
puts "The first time loading Clio can take several minutes, as caches need to be built."
```

File: exe/clio_setup [hunk 4]
```diff
@@ -64,4 +64,30 @@ STEPS.each do |klass|
...
+puts '              ;     |         \      /'
+puts '               \/ ,/           \    |'
+puts '               / /~~|~|~~~~~~|~|\   |'
+puts '              / /   | |      | | `\ \ '
+puts '             / /    | |      | |   \ \ '
+puts '            / (     | |      | |    \ \ '
+puts '      |||| /,_)||| /__) ||| /__)|| /,_/ |||'
+puts '     """"""""""""""""""""""""""""""""""""""'
+puts "\r\r\rWelcome home."
+puts "\r\rPlease restart your terminal for the appropriate changes to take effect."
```
Comment (fimmtiu) [inline exe/clio_setup]: How about this wording?
```suggestion
puts "\r\rPlease close and re-open your terminal to ensure that all changes take effect."
```
Oh man, this suggestions feature is hella sweet.

File: exe/clio_setup [hunk 5]
```diff
@@ -64,4 +64,30 @@ STEPS.each do |klass|
   end
 end
 
+puts "\r\r"
```
Comment (fimmtiu) [inline exe/clio_setup]: I'm guessing you didn't test this out, because `\r` does not do what you think it does. :-)

File: exe/clio_setup (hunk 1 above)
Comment (jamescgoertzen) [reply exe/clio_setup]: 🐐

File: exe/clio_setup (hunk 5 above)
Comment (jamescgoertzen) [reply exe/clio_setup]: Haha it does sometimes 😭

File: exe/clio_setup (hunk 4 above)
Comment (jamescgoertzen) [reply exe/clio_setup]: Haha it's pretty sweet, but it makes me feel lazy because I can just apply everything 😉
